schemas/configuration/v5/server_config.json.in

```diff
@@ -80,13 +131,12 @@
             },
             "required": ["control_type", "users"]
         },
-        "maximum_connections": {
-            "type": "integer",
-            "minimum": 1
-        },
         "default_dir_mode": {"type": "string"},
         "default_file_mode": {"type": "string"},
```

Comment:
Should these be locked at a length of 4? e.g. "0600", "0750"

Or, it could be an integer with a minimum value of 0 and a maximum value of 777, but then we have to modify the code and upgrade to account for the type switch. Ehhh...

Reply:
I think we can do either approach. If json schema supports octal values, then going the integer route seems best, but also seems like the easiest way to brick 5.0. So, changing to an integer feels like a iRODS 6 thing (due to timing).

The max length solution sounds like the safest path forward.

Reply:
Updated the schema to require that these properties have an exact length of 4.